app/views/admin/project_logs/_log_list.html.slim

```diff
@@ -1,18 +1,21 @@
-
 - if @logs.empty?
   = content_tag(:div, t('common.no_records'), class: 'alert alert-info')
 - else
   .show-actions.hidden-print
     a data-expand-all=".logs-index" href="#" = t('common.expand_all')
-    a data-hide-all=".logs-index" href="#" = t('common.hide_all')
+    - if @step && @context && @context == "calendar"
```

Comment:
Also, what is the purpose of checking `@step`? I see a mixture of if statements here, some checking `@step`, some not. What happens if we change them all to just `@context == "calendar"`? That would be much clearer...